ci/kokoro/windows/build-32.bat

```diff
@@ -21,8 +21,13 @@ REM Install Bazelisk.
 REM Change PATH to install the Bazelisk version we just installed
 @set PATH=C:\bin;%PATH%
 
+@REM Before MSVC 2022 the compiler is a 32-bit binary
 REM Configure the environment to use MSVC %MSVC_VERSION% and then switch to PowerShell.
-call "c:\Program Files (x86)\Microsoft Visual Studio\%MSVC_VERSION%\Community\VC\Auxiliary\Build\vcvars32.bat"
+if %MSVC_VERSION% GEQ 2022 (
+  call "c:\Program Files\Microsoft Visual Studio\%MSVC_VERSION%\Community\VC\Auxiliary\Build\vcvars64.bat"
+) else (
+  call "c:\Program Files (x86)\Microsoft Visual Studio\%MSVC_VERSION%\Community\VC\Auxiliary\Build\vcvars64.bat"
```

Comment:
should this be s/vcvars64.bat/vcvars32.bat/ ?

Reply:
Thanks.